apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/g_Environment/EnvironmentVariables.py

```
@@ -0,0 +1,75 @@
+# System Imports #
+import os
+
+# ------------------------------------------ Database ------------------------------------------ #
+
+# Database start configuration #
+os.environ['main_db_conn'] = 'pgsql'
```

Ruff (0.11.9)

Use capitalized environment variable `MAIN_DB_CONN` instead of `main_db_conn`

Replace `main_db_conn` with `MAIN_DB_CONN`

(SIM112)